docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,69 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table)
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+
+Image Support
+-------------
+
+- [ ] Flash an uncompressed image
+- [ ] Flash a Bzip2 image
+- [ ] Flash a XZ image
+- [ ] Flash a ZIP image
+
+Flashing Process
+----------------
+
+- [ ] Close the application while flashing using the window manager close icon
+- [ ] Close the application while flashing using the OS keyboard shortcut
+- [ ] If applicable, close the application from the terminal using Ctrl-C while
+  ashing
+- [ ] Force kill the application (using a process monitor tool, etc)
+- [ ] Cancel the application close warning dialog
+
+In all these cases, the child writer process should not remain alive.
```

Comment:
Might be worth adding a note that the child writer process is running as root / admin (as some tools like top/htop etc need to be run as root as well to see those processes)

Reply:
Is that really the case? I just checked and I can see processes belonging to other users.

Reply:
That's odd I can't see the elevated writer process if I'm not sudo-ing

Reply:
Weird. I added a note just in case.